sumpy/expansion/__init__.py

```diff
@@ -678,36 +629,19 @@ def __init__(self, kernel, order, use_rscale):
         self.expansion_terms_wrangler_key = (order, kernel.dim)
 
 
-class LaplaceConformingVolumeTaylorExpansion(VolumeTaylorExpansionBase):
+class LinearPDEConformingVolumeTaylorExpansion(VolumeTaylorExpansionBase):
 
-    expansion_terms_wrangler_class = LaplaceExpansionTermsWrangler
+    expansion_terms_wrangler_class = LinearPDEBasedExpansionTermsWrangler
     expansion_terms_wrangler_cache = {}
 
     # not user-facing, be strict about having to pass use_rscale
     def __init__(self, kernel, order, use_rscale):
-        self.expansion_terms_wrangler_key = (order, kernel.dim)
-
+        self.expansion_terms_wrangler_key = (order, kernel.dim, kernel)
 
-class HelmholtzConformingVolumeTaylorExpansion(VolumeTaylorExpansionBase):
-
-    expansion_terms_wrangler_class = HelmholtzExpansionTermsWrangler
-    expansion_terms_wrangler_cache = {}
-
-    # not user-facing, be strict about having to pass use_rscale
-    def __init__(self, kernel, order, use_rscale):
-        helmholtz_k_name = kernel.get_base_kernel().helmholtz_k_name
-        self.expansion_terms_wrangler_key = (order, kernel.dim, helmholtz_k_name)
-
-
-class BiharmonicConformingVolumeTaylorExpansion(VolumeTaylorExpansionBase):
-
-    expansion_terms_wrangler_class = BiharmonicExpansionTermsWrangler
-    expansion_terms_wrangler_cache = {}
-
-    # not user-facing, be strict about having to pass use_rscale
-    def __init__(self, kernel, order, use_rscale):
-        self.expansion_terms_wrangler_key = (order, kernel.dim)
 
+LaplaceConformingVolumeTaylorExpansion = LinearPDEConformingVolumeTaylorExpansion
+HelmholtzConformingVolumeTaylorExpansion = LinearPDEConformingVolumeTaylorExpansion
+BiharmonicConformingVolumeTaylorExpansion = LinearPDEConformingVolumeTaylorExpansion
```

Comment:
Should these complain that they're deprecated? (Like make a subclass with a complain-y constructor and then derive these from that?)